two-factor.php

```diff
@@ -248,14 +248,14 @@ function wpcom_vip_two_factor_admin_notice() {
 			<div class="dashicons dashicons-warning" style="display:flex;float:left;margin-right:2rem;font-size:38px;align-items:center;margin-left:-20px;color:#ffb900;"></div>
 			<div>
 				<p style="font-weight:bold; font-size:16px;">
-					<a href="https://wpvip.com/documentation/vip-go/two-factor-authentication-on-vip-go/">Two Factor Authentication</a> is required to edit content on this site.
+					Access requires your account to have <a href="https://wpvip.com/documentation/vip-go/two-factor-authentication-on-vip-go/">Two-factor authentication</a> enabled.
 				</p>
 
-				<p>For the safety and security of this site, your account access has been downgraded. Please enable two-factor authentication to restore your access.</p>
+				<p>For the safety and security of this site, your account access has been downgraded. Please enable Two-factor authentication to restore your access.</p>
 
 				<p>
 					<a href="<?php echo esc_url( admin_url( 'profile.php#two-factor-options' ) ); ?>" class="button button-primary">
-						Enable Two-factor Authentication
+						Enable Two-factor authentication
```

Comment:
```suggestion
						Enable two-factor authentication
```

Reply:
Since it's a button, I feel it's better to have T in uppercase

Reply:
In which case the A should be as well.

Reply:
That's true. Updated in [`4e18d4b`](https://github.com/Automattic/vip-go-mu-plugins/pull/5133/commits/4e18d4bef0450b7f403ad8c8d28251d9f59793ad)